x-pack/legacy/plugins/spaces/public/views/management/edit_space/customize_space/customize_space_avatar.tsx

```diff
@@ -55,6 +128,7 @@ class CustomizeSpaceAvatarUI extends Component<Props, State> {
             // without defaulting to the derived initials provided by `getSpaceInitials`
             value={initialsHasFocus ? pendingInitials || '' : getSpaceInitials(space)}
             onChange={this.onInitialsChange}
+            disabled={this.props.space.imageUrl && this.props.space.imageUrl !== '' ? true : false}
           />
         </EuiFormRow>
         <EuiSpacer size="m" />
```

Comment:
This `EuiSpacer` on line 134 and the one on line 148 can be removed. 

Other than that, it looks great!

Reply:
@ryankeairns : that EuiSpacer was added by @cchaos, I added the one on line 148 for uniformity. If you check the master branch, it's there. Am I right?

Reply:
Its definitely too much space, but there has been some other work going on around form inputs so perhaps something changed. I'll add my approval and I will come by later to make design changes in a separate PR. We're currently starting a form clean up effort across Kibana, so I'll touch this up. Thanks!

Reply:
@ryankeairns : did you clean up your browser cache or align your code to the master branch? I don't see all that space when I run Kibana.

Reply:
@ryankeairns I'm not seeing that either. Am I overlooking something?

![image](https://user-images.githubusercontent.com/3493255/65053606-301c4380-d93a-11e9-96a1-5d107501ce1c.png)

Reply:
Just removed my local branch and pulled this PR down fresh, all clear!

<img width="329" alt="Screenshot 2019-09-17 10 53 40" src="https://user-images.githubusercontent.com/446285/65058221-8341c680-d939-11e9-8bc3-5e716117917d.png">